ios/CodePush/RCTConvert+CodePushInstallMode.m

```diff
@@ -1,5 +1,10 @@
 #import "CodePush.h"
+
+#if __has_include("RCTConvert.h")
```

Comment:
> I could hypothesize that both "RCTConvert.h" and <React/RCTConvert.h> exist in 0.40

actually that is what i thought too, but `find node_modules/ -name RCTBridge*` finds only occurrence. Also even if there were two of them, I though that #if / #else is safe enough to not allow importing file more than once.

I saw this commit, where you're changing order and I though, you had same issue with duplicate imports, but if it's just cosmetic, then it remains mystery for me.

Anyway thank you for time and answer.